articles/Silo_Finance.md

```diff
@@ -0,0 +1,388 @@
+
+As with other stablecoins, keeping the peg depends on a few key factors:
+
+
+
+1. The underlying collateral needs to be fit for purpose (i.e. highly volatile collateral increases the risk of default and the risk of liquidations)
+2. For the market to absorb liquidations, as well as arbitrage to keep the peg, both the stablecoin and the collateral asset need to have deep enough liquidity
+3. The protocol can support price stability via parameter adjustments
+4. Ideally, the protocol has some fallback solution in case of a black swan event
+
+We’ll discuss the collateral risk (1) in more detail in the next section. The other stabilization mechanisms highlighted above are mostly covered:
+
+
+
+* XAI is an over-collateralized stablecoin. At first only USDC and ETH will serve as collateral for XAI. As the usage of XAI expands to other pools, this will change and XAI can be backed by more volatile assets too
+* Arbitrage possibilities are given and liquidity in the open market will be seeded by the DAO and incentivized through CVX gauges (see [proposal](https://gov.silo.finance/t/building-on-chain-liquidity-for-xai/309))
+* The DAO influences XAI via parameter adjustments, such as borrow rate, adding/removing collateral, increasing/decreasing XAI availability
+
+The only measure that is missing is a concrete fallback solution, that the DAO can execute in case things go south (e.g. use SILO as a backup to stabilize the stablecoin). However, this can also come at a later stage, once XAI has achieved some product market fit.
+
+The last thing to highlight related to stability is that Silo’s TVL is currently around ~$2.6M. However, most of that TVL was [seeded by the DAO](https://snapshot.org/#/silofinance.eth/proposal/0x013bbb154af9317ab7a78b72a498d16e069fdba5e78b710e6785fc938d644e76) itself. There needs to be significant growth in TVL to enable sufficient backing of a stablecoin that aims to facilitate bridging between 60+ assets (10 assets are already live and [57 assets](https://gov.silo.finance/t/deploying-57-new-silos-to-the-protocol/306/2) will be added soon). While there are plans to incentivize liquidity for XAI, there is currently no plan presented that defines anything specific about how the DAO plans to attract more TVL for its long-tail asset silos (a potential [veTokenomics](https://gov.silo.finance/t/tokenomics-proposal-vesilo-v2/226) discussion seems to be stuck).
+
+
+## Collateral Risk
+
+In the beginning, the only collateral enabled to borrow XAI will be USDC and ETH. Thus, eliminating other risks that come with long tail assets (e.g., low asset liquidity, high price volatility, depeg of the collateral, etc.).
+
+Both ETH and USDC are highest-tier assets with more than enough liquidity to offset potential liquidations. Even when XAI will be enabled as a bridge asset for more silos, the collateral risk is limited only to those isolated markets.
+
```

Comment:
**the** highest-tier or maybe just "quality"

Reply:
true, much better